main.cpp

```diff
@@ -223,12 +296,28 @@ struct Teacher
     Teacher();
     //1) give lecture
     void giveLecture();
-
     //2) private tutoring
     // returns the amount of money made from private tutoring
     float privateTutoring(float hourlyRate, int numberOfHours = 1); //input hourly rate and number of hours
     //3) assign homework
     void assignHomework(std::string homeworkAssignment);//input a string representation of the homework
+
+    void sayMeaninglessNumbers(int startingWith)
+    {
+        if(startingWith > 0)
+        {        
+            int numberOfStuff = numberOfClassesTaught * startingWith;
+            std::cout << "I have taught " << numberOfStuff << "things" << std::endl;
+            std::cout << "that is " ;
+            while(startingWith > 0)
+            {
+                std::cout << --startingWith;
```

Comment:
separate math expressions from usage of variables in std::cout statements
it will improve the readability and ambiguity with regard to when the decrement happens
```
--startingWith;
std::cout << startingWith << " ";
- [ ] revise